.expeditor/verify.pipeline.yml

```diff
@@ -2,7 +2,9 @@ expeditor:
   defaults:
     buildkite:
       timeout_in_minutes: 60
-
+      env: 
+        HAB_BLDR_CHANNEL: "LTS-2024"
+        HAB_INTERNAL_BLDR_CHANNEL: "LTS-2024"
```

Comment:
You shouldn't need to set `HAB_INTERNAL_BLDR_CHANNEL` in this file. That is used when you using a non-stable version of the hab binary and you also need matching non-stable versions of the other hab components like hab-sup and studio.